netclient/ncutils/netclientutils.go

```diff
@@ -515,6 +515,22 @@ func CheckUID() {
 	}
 }
 
+// CheckFirewall - checks if iptables of nft install, if not exit
+func CheckFirewall() {
+	found := false
+	_, err := exec.LookPath("iptables")
+	if err == nil {
+		found = true
+	}
+	_, err = exec.LookPath("nft")
+	if err == nil {
+		found = true
+	}
+	if !found {
+		log.Fatal("neither iptables nor nft is installed - please install one or the other and try again")
```

Comment:
@mattkasun does this mean it will be mandatory to have either iptables or nft installed in order to run the netclient? Up until this point, I believe we've been able to run the netclient without iptables, and only needed it if doing ingress/egress.

Reply:
currently yes, other option is to add a new field to nodes... canBeGateway or something similar .. would be set on startup and be limited to freebsd and linux with iptables or nft.   If flag is false - gateway creation in UI is disabled for that node

Reply:
@afeiszli we can leverage the firewall field on nodes and put an option as invalid and accomplish what matt is saying and then deny gateway creations on nodes that are invalid

Reply:
updated to match @0xdcarns comment